tests/fixtures/apps/balsamic/case/config.json

```diff
@@ -183,26 +183,36 @@
         "BALSAMIC_version": "8.2.5",
         "config_creation_date": "2021-12-28 13:12"
     },
-    "samples": {
-        "concatenated_ACC0000A0_XXXXXX_R": {
-            "file_prefix": "concatenated_ACC0000A0_XXXXXX_R",
-            "sample_name": "ACC0000A0",
+    "samples": [
+        {
             "type": "normal",
-            "readpair_suffix": [
-                "1",
-                "2"
-            ]
+            "name": "ACC0000A0",
+            "fastq_info": {
+                "H3CVVDRXY_ACC0000A0_S68_L001": {
+                    "fwd": "/mock/path/cases/mock_case/analysis/fastq//H3CVVDRXY_ACC0000A0_S68_L001_R1_001.fastq.gz",
```

Comment:
Can we use Path and `as_posix` instead to not hard code ´/´

Reply:
We can't do that since it's a JSON file. However, this suggests that it should probably be a fixture... I will try to change it when I have some time to refactor the Balsamic analysis API. Thanks for the comments and review!